### DISCUSSION TITLE
What is in a name? Commonhaus Foundation is (mostly) a placeholder

### DISCUSSION THREAD

**Discussion by ebullient** [commonhaus/foundation#27] [Ideas]
I will confess that having written and mangled many documents to use `Commonhaus Foundation` it has become the thing, but it was intended as a placeholder.

We should have other names handy... 

Some that were thrown around earlier (yes, Codehaus folks are here.. )

- Grinder
- Roasthaus
- AbstractProjectFactory
- OpenForever Foundation

Please pitch in! No bad ideas!

**Comment by angellk** [comment]
So... I would recommend against 'Grinder'. What's a name that truly embodies what this foundation does that others don't? 'AbstractProjectFactory' may be closest .. and yet .. maybe too abstract?

**Reply by ebullient** [reply]
In that case.. Grinder was specifically a reference to all things coffe and beans, etc.

**Reply by angellk** [reply]
I figured.. my recommendation against would be that it's too close to a certain dating app. I looked at all the types of coffee and it looks like everyone has an open source project associated :(

**Comment by sigrunixia** [comment]
Jokingly... BBEC, Bountiful [and] Beautifully Exciting Code. 

More seriously: 
- We'reOpen
- OpenMingle Latte
- CodeCrafters Cafe
- ByteHub
- ByteHaus

**Comment by kenfinnigan** [comment]
What about "NoSugar"? It's a reference to coffee, but also no additives, just getting things done
